### ISSUE TITLE
Vantage deep links

### ISSUE THREAD

**Issue by roborourke** [humanmade/altis-documentation#30]
Like #25 - a way to link directly to a section of Vantage from the docs would be super neat. Low priority as not many use cases yet.

**Comment by rmccue** [comment]
Question: what do we gain here over just linking directly to the URL? Guessing the knowledge of which stack we're running on, but we don't currently have that for local anyway, which is presumably where it would be most useful.

**Comment by roborourke** [comment]
@rmccue yeah though it's not not useful on staging/dev/production sites, at least right now as they point to individual stacks.

Ideally we need stack grouping eg. by repo and we just document how to add your stack ID to the composer config.

Alternatively we add a way for vantage to resolve a stack or group of stacks by the associated repo URL. Guess that'll need to happen before this can though.

**Comment by rmccue** [comment]
I know @grarighe's original designs for Vantage had stacks grouped by project, so maybe we'll get that done?

But with that said, there's separate deploys/logs/etc pages per stack, so we can't deep link to those pages without a specific stack.

**Comment by roborourke** [comment]
Very little benfit to this as we may remove the docs in the admin at some point anyway in favour of pointing them to the docs site.
